discovery-provider/src/utils/helpers.py

```diff
@@ -481,3 +481,28 @@ def get_tx_arg(tx, arg_name):
 def split_list(list, n):
     for i in range(0, len(list), n):
         yield list[i : i + n]
+
+
+# Extracts the pre and post balances for a given index from a solana transaction metadata
+def get_solana_tx_balances(meta: Any, idx: int) -> Tuple[int, int]:
```

Comment:
love this. does the comment get included in autocomplete/intellisense? I think if you put it in triple quotes within the body it does fwiw.